src/microbe_stage/Microbe.Contact.cs

```diff
@@ -505,14 +505,19 @@ public void Kill()
             }
         }
 
-        int chunksToSpawn = Math.Max(1, HexCount / Constants.CORPSE_CHUNK_DIVISOR);
+        // Queues either 1 corpse chunk or a factor of the organelles
+        int chunksToSpawn = Math.Max(1, organelles.Count / Constants.CORPSE_CHUNK_DIVISOR);
 
         var chunkScene = SpawnHelpers.LoadChunkScene();
 
+        // Duplicates the organelle list in a random order and creates an enumerator from it
```

Comment:
You should write in the comment *why* something is done, not just what is being done here because someone reading this code will be able to figure out what this does but they can't figure out why it was done.